front_end/WorkoutWise/src/components/Navbar.jsx

```diff
@@ -1,19 +1,33 @@
 import { Link } from "react-router-dom";
-export default function Navbar() {
+import { useState, useContext } from "react";
+import { userContext } from "../App";
+import { api } from "../utilities";
+import { useNavigate } from "react-router-dom";
 
+export default function Navbar() {
+    const { logout, user } = useContext(userContext);
+    const [ searchInput, setSearchInput ] = useState("");
+    
+    const onChangeHandler =(event) => {
+        setSearchInput(event.target.value);
+    };
     return (
-    <>
-        (<nav>
-            <h1>WorkoutWise</h1>
-            <Link>Home</Link>
-            <Link>Workout</Link>
-            <input type="text" placeholder="search" />
-            <button >
-            <Link >
-                Search
-            </Link>
-            </button>
-        </nav>)
-    </>
+        <>
+            <nav>
+                <h1>WorkoutWise</h1>
+                {/* <Link to="/">Home  </Link> */}
+                <Link to={"workouts"}>Workout   </Link>
+                <Link to={"myexercises"}>Exercises   </Link>
+                <input type="text" placeholder="search" onChange={onChangeHandler} value={searchInput}/>
```

Comment:
Maybe this could be a form that triggers the `useNavigate` hook on submit with the proper parameters. This current set up forces the user to click the button and takes away their ability to press enter for submitting their input.